src/and-cli-workspace.ts

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env node
```

Comment:
@brandongregoryscott I can't quite spot the breaking change without digging in further, but I am noticing a regression with the `workspace` command's exiting. No matter the variation (even `--help`) it hangs and I can't even `ctrl+c` to exit.

Thing is... it eventually does exit. Just takes _several minutes_.

Reply:
Weird. I've been testing this out this morning and I did not see the slowness with the `github` command, but I did with the `workspace` command. I even stripped out almost everything (functions, imports, etc) from that command and it still seems to hang for ~30s to display the help menu... 🤔